widgetssdk/src/main/java/com/glia/widgets/GliaWidgetsConfig.java

```diff
@@ -14,18 +14,6 @@
     private final String region;
     private final int requestCode;
 
-    /**
-     * Deprecated since SDK version 1.6.5. Please use {@link GliaWidgetsConfig#GliaWidgetsConfig(String, String, Context, String, int)} instead.
-     */
-    @Deprecated
-    public GliaWidgetsConfig(String appToken, String apiToken, String siteId, Context context, String region, int requestCode) {
```

Comment:
Do I understand correctly that all integrators will get a compilation error in case of removing this initializer? If so, let's avoid making breaking changes and let it as is with deprecation note.

Reply:
yup, hence the 1.7 update

Reply:
- returned the constructor and setter with the deprecation notice